src/index.html

```diff
@@ -4,9 +4,42 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/style.css">
   </head>
   <body>
     <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
You are not making any use of modifiers. They should determine What is a starting day and how many days should be displayed. Please see the readme 🙏